modules/dnn/test/test_common.impl.hpp

```diff
@@ -217,6 +221,16 @@ testing::internal::ParamGenerator< tuple<Backend, Target> > dnnBackendsAndTarget
         for (std::vector< Target >::const_iterator i = available.begin(); i != available.end(); ++i)
             targets.push_back(make_tuple(DNN_BACKEND_VKCOM, *i));
     }
+
+#ifdef HAVE_CUDA
+    if(withCUDA)
+    {
+        //for (auto target : getAvailableTargets(DNN_BACKEND_CUDA))
+        //    targets.push_back(make_tuple(DNN_BACKEND_CUDA, target));
```

Comment:
please remove commented code

Reply:
I'll need more time for this. FP16 inference of ENet crashes on Linux but doesn't crash on Windows. Need to investigate.